src/interactive.c

```diff
@@ -71,7 +71,7 @@ interactive_begin(struct view *view, enum input_mode mode, uint32_t edges)
 		cursor_set(seat, LAB_CURSOR_GRAB);
 		break;
 	case LAB_INPUT_STATE_RESIZE:
-		if (view->maximized || view->fullscreen) {
+		if (view->fullscreen || view->maximized != VIEW_AXIS_NONE) {
 			/*
 			 * We don't allow resizing while maximized or
 			 * fullscreen.
```

Comment:
I am not sure about this. I'd assume people would actually want to resize in a `maximized != VIEW_AXIS_BOTH` situation.

Reply:
For now we could do the same as resizing a tiled window: you can resize it but then it loses the tiled/maximized state.

Reply:
Pushed a [fixup] commit to make resizing work similar to how it does in tiled states.